source/plugins/go/src/extension/extension.go

```diff
@@ -64,7 +64,7 @@ func getDataTypeToStreamIdMapping() (map[string]string, error) {
 	}
 
 	fs := &FluentSocket{}
-	fs.sockAddress = "/var/run/mdsd/default_fluent.socket"
+	fs.sockAddress = "/var/run/mdsd-ci/default_fluent.socket"
 	if containerType != "" && strings.Compare(strings.ToLower(containerType), "prometheussidecar") == 0 {
 		fs.sockAddress = fmt.Sprintf("/var/run/mdsd-%s/default_fluent.socket", containerType)

```

Comment:
same here. can you please double check on the socket names for both main and sidecar containers?

Reply:
the code changes are correct. the new paths will be /var/run/mdsd-ci/ and /var/run/mdsd-PrometheusSidecar